backend/geonature/core/gn_meta/routes.py

```diff
@@ -259,13 +253,12 @@ def uuid_report():
     id_import = params.get("id_import")
     id_module = params.get("id_module")
 
-    query = DB.session.query(Synthese).select_from(Synthese)
-
-    if id_module:
-        query = query.filter(Synthese.id_module == id_module)
-
-    if ds_id:
-        query = query.filter(Synthese.id_dataset == ds_id)
+    query = (
+        DB.select(Synthese)
+        .select_from(Synthese)
+        .where_if(id_module, Synthese.id_module == id_module)
+        .where_if(ds_id, Synthese.id_dataset == ds_id)
```

Comment:
```suggestion
        .where_if(ds_id is not None, Synthese.id_dataset == ds_id)
```

Reply:
C'est pas très conseillé le `is not None`.